projection/core/src/main/scala/com/lightbend/lagom/internal/projection/ProjectionRegistryActor.scala

```diff
@@ -91,7 +91,7 @@ class ProjectionRegistryActor extends Actor with ActorLogging {
   // required to handle Terminate(deadActor)
   var reversedActorIndex: Map[ActorRef, WorkerKey] = Map.empty[ActorRef, WorkerKey]
 
-  val DefaultInitialStatus: Status = {
+  val DefaultRequestedStatus: Status = {
```

Comment:
The `lagom.projection.auto-start.enabled` setting only applies to `requetedStatus`.